### PR TITLE
[t6] Add HTC One Max to fixup-mountpoints

### PR DESCRIPTION
**Pull request by rltyseven**

### CODE REVIEW

File: fixup-mountpoints [hunk 1]
```diff
@@ -533,343 +44,11 @@ case "$DEVICE" in
...
-            -e 's block/bootdevice/by-name/modemst2 mmcblk0p14 ' \
-            -e 's block/bootdevice/by-name/persist mmcblk0p26 ' \
-            -e 's block/bootdevice/by-name/recovery mmcblk0p22 ' \
-            -e 's block/bootdevice/by-name/rpm mmcblk0p6 ' \
-            -e 's block/bootdevice/by-name/sbl1 mmcblk0p4 ' \
-            -e 's block/bootdevice/by-name/splash mmcblk0p18 ' \
-            -e 's block/bootdevice/by-name/ssd mmcblk0p3 ' \
-            -e 's block/bootdevice/by-name/system mmcblk0p24 ' \
-            -e 's block/bootdevice/by-name/userdata mmcblk0p49 ' \
-            -e 's block/bootdevice/by-name/oem mmcblk0p30 ' \
```
Comment (sledges) [inline fixup-mountpoints]: Please do not delete devices that you do not have, they are there for others and they do not interfere.

Please avoid creating a pull request from your master branch, as I think this commit was not meant for this PR.

File: fixup-mountpoints [hunk 2]
```diff
@@ -420,23 +420,28 @@ case "$DEVICE" in
...
+            -e 's block/platform/msm_sdcc.1/by-name/cache mmcblk0p41 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/misc mmcblk0p21 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/radio mmcblk0p34 ' \
             -e 's block/platform/msm_sdcc.1/by-name/recovery mmcblk0p38 ' \
             -e 's block/platform/msm_sdcc.1/by-name/system mmcblk0p39 ' \
-            -e 's block/platform/msm_sdcc.1/by-name/modem_st1 mmcblk0p22 ' \
-            -e 's block/platform/msm_sdcc.1/by-name/modem_st2 mmcblk0p23 ' \
-            -e 's block/platform/msm_sdcc.1/by-name/cache mmcblk0p41 ' \
             -e 's block/platform/msm_sdcc.1/by-name/userdata mmcblk0p42 ' \
+            -e 's block/platform/msm_sdcc.1/by-name/wcnss mmcblk0p18 ' \
```
Comment (sledges) [inline fixup-mountpoints]: Could you please squash this commit into the main one (the main commit should be the only commit in this PR)

**Comment by rltyseven** [comment]
Has been re-fixed.

File: fixup-mountpoints (hunk 2 above)
Comment (rltyseven) [reply fixup-mountpoints]: Has been re-fixed.

File: fixup-mountpoints (hunk 1 above)
Comment (rltyseven) [reply fixup-mountpoints]: Has been re-fixed.

**Comment by sledges** [comment]
There is still a merge commit, could you please cherry-pick the t6 commit on top of the master branch cleanly?
Ask for help on #sailfishos-porters channel if needed

**Comment by rltyseven** [comment]
no problem. In fact I saw dmesg yesterday, need to make some changes.

**Pull request closed without merging**